### ISSUE TITLE
Spacing around image captions can be improved

### ISSUE THREAD

**Issue by timvink** [squidfunk/mkdocs-material#1940]
__I checked that...__

- [x] ... the documentation does not mention anything about my problem
- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... there are no open or closed issues that are related to my problem

## Description

When you add two images with captions below each other, the top caption looks like it belong to the bottom one due to spacing. Let me illustrate that with some cats :)

![image](https://user-images.githubusercontent.com/5570380/93884198-dedc2800-fce2-11ea-9bea-d93496340c22.png)

### Expected behavior

I expect to have the caption closer to the image it belongs to. So the top padding should be slightly smaller than the bottom padding:

![image](https://user-images.githubusercontent.com/5570380/93884297-04693180-fce3-11ea-99a4-5cfeba987a30.png)

### Steps to reproduce the bug

```mkdocs.yml
# mkdocs.yml
site_name: Test

theme:
    name: material
```
```html
<!-- index.md -->
# A page

with images

![A cat playing 1](https://play-lh.googleusercontent.com/JHDqEqU0QNC8vsa5_UsPAws5X1OvTVPcfDVLnV1WXhoYrEX81sE6fL7cmamStPrK_A=w1440-h620-rw)
![Another cat playing](https://play-lh.googleusercontent.com/tKuOgQBwDTjhZg3DjCdZVTSVa9X9iMrtrM_1JDH6Ky_YyQeKw_bnFDy0tj1aZ39TDnI=w1440-h620-r)
![More cats](https://play-lh.googleusercontent.com/4CPahw1_E0b61tZKq4QI1bw_dqR6bYy0aDiNWrn-MCoz9Wq5bNyhKywfVlK01nNKR-A=w1440-h620-r)
```

### Package versions

* Python: 3.7.4
* MkDocs: version 1.1.2
* Material:  5.5.12
* OS: MacOS
* Browser: Chrome

**Comment by squidfunk** [comment]
Fixed in 01f1aed9.

**Comment by squidfunk** [comment]
Released as part of 5.5.14.
